pyproject.toml

```diff
@@ -31,11 +31,11 @@ classifiers = [
 ]
 requires-python = ">=3.10"
 readme = "README.md"
-license = { text = "MPL 2.0" }
+license = "MPL-2.0"
```

Comment:
[License should be a SPDX string from metadata 2.4+](https://packaging.python.org/en/latest/specifications/pyproject-toml/#license).